# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLExtensionCompressedTextureS3TC.cpp
# Commit: ccfa767dba64
# Full Hash: ccfa767dba644dc193e0246eb8e8ff3377e8b8a5
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Client-side bindings mirror for precise CC, and merge similar codepaths. r=handyman
#   
#   * Context loss using RAII
#   * Move Program reflection Client-side
#   
# ==============================================================================

diff -r 71c122ac0ca7 -r ccfa767dba64 dom/canvas/WebGLExtensionCompressedTextureS3TC.cpp
--- a/dom/canvas/WebGLExtensionCompressedTextureS3TC.cpp	Wed Jan 08 22:19:14 2020 +0000
+++ b/dom/canvas/WebGLExtensionCompressedTextureS3TC.cpp	Wed Jan 08 22:19:16 2020 +0000
@@ -22,8 +22,6 @@
     auto usage = fua->EditUsage(effFormat);
     usage->isFilterable = true;
     fua->AllowSizedTexFormat(sizedFormat, usage);
-
-    webgl_->mCompressedTextureFormats.AppendElement(sizedFormat);
   };
 
 #define FOO(x) LOCAL_GL_##x, webgl::EffectiveFormat::x
@@ -36,8 +34,6 @@
 #undef FOO
 }
 
-WebGLExtensionCompressedTextureS3TC::~WebGLExtensionCompressedTextureS3TC() {}
-
 bool WebGLExtensionCompressedTextureS3TC::IsSupported(
     const WebGLContext* webgl) {
   gl::GLContext* gl = webgl->GL();